praw/models/user.py

```diff
@@ -156,8 +156,8 @@ def me(
         if self._reddit.read_only:
             if not self._reddit.config.custom.get("praw8_raise_exception_on_me"):
                 warn(
-                    "The `None` return value is deprecated, and will raise a"
-                    " `ReadOnlyException` beginning with PRAW 8. See documentation for"
+                    "The 'None' return value is deprecated, and will raise a"
```

Comment:
This is precisely the case that I think makes using `'` weird since, `None` is not a string. In many of the other locations the quoted values are strings. To be pedantic, we might want to highlight types in the RTD docs, which means `None` should appear as `\`\`None\`\``. And strings maybe should appear as `\`\`"SOMESTRING"\`\``.

Reply:
> To be pedantic, we might want to highlight types in the RTD docs, which means None should appear as:
> 
> ``None``
And strings maybe should appear as:
> 
> ``"SOMESTRING"``
> So that they render in RTD as None and "SOMESTRING". But this is rather complicated.

This was already done to the docs.